pkg/provisioner/ironic/ironic.go

```diff
@@ -1167,7 +1167,7 @@ func (p *ironicProvisioner) buildManualCleaningSteps(bmcAccess bmc.AccessDetails
 	var newSettings []map[string]interface{}
 	if data.ActualFirmwareSettings != nil {
 		// If we have the current settings from Ironic, update the settings to contain:
-		// 1. settings converted by BMC drivers that are different than current settings
+		// 1. settings converted by BMC drivers that are different from current settings
```

Comment:
"different than" is correct and certainly not an error. Although there is some discussion about which is the preferred phrasing, see for example https://www.merriam-webster.com/words-at-play/different-from-or-different-than, it hardly seems worth changing.

Reply:
Thank you for clarifying that. I will close this PR.